node_addons/FormatConverter/setenv.sh

```diff
@@ -1,118 +1,149 @@
 # !/bin/bash
 
-[ "$1" = "DEBUG" ] && build_conf=Debug || build_conf=Release
-
-[ "$build_conf" != "Debug" ] && echo "target config: release" ||  echo "target config: debug"
+ROOT_PATH=`dirname ${BASH_SOURCE[0]}`
+echo "ROOT_PATH=$ROOT_PATH"
+FFMPEG_DOWNLOAD_PATH=~/
+# for Darwin recommended:
+# FFMPEG_DOWNLOAD_PATH=~/Documents/
+BIN_PATH=$ROOT_PATH/../../bin
+FFMPEG_DEV_PATH=
+FORMAT_CONVERTER_BIN=FormatConverter.so
+PREPDIR="/opt/kaltura/builds/${APP_NAME}"
+BUILD_CONF=Release
+
+while [ "$1" != "" ]; do
+	key="$1"
+
+    case $key in
+        -p | --path )
+                    shift
+			        BIN_PATH=$1
+			        #[ ! -e $1 ] && $BIN_PATH=$PREPDIR/$FFMPEG_BIN_DIR
+			        echo "-p | --path $1"
+			        shift
+        ;;
+        -m | --mode )
+                    shift
+                    [ $1 = 'debug' ] && BUILD_CONF=Debug || BUILD_CONF=Release
+			        echo "-m | --mode $1"
+			        shift
+		;;
+        -h | --help )
+			        echo "-p | --path [full path], FormatConverter bin path"
```

Comment:
why give a different format convertor path?

the idea is to give FFMPEG source location